manifests/stub.pp

```diff
@@ -33,6 +33,7 @@
   $insecure    = false,
   $type        = 'transparent',
   $config_file = $unbound::params::config_file,
+  $stub_first = 'no',
```

Comment:
```suggestion
  $stub_first  = 'no',
```
Only for alignment :laughing:

Reply:
Maybe you could go for false instead of 'no'.
Then in stub.erb:

```
<% if @stub_first -%>
...
```

Reply:
It's done on purpose. Default is 'no', so we don't add anything. If you want to add it, specify it exactly, otherwise, it will still be no.